### ISSUE TITLE
Preferred BPM

### ISSUE THREAD

**Issue by rodrigonsh** [mixxxdj/mixxx#13047]
### Feature Description

Some songs come too fast or too slow depending on the DJ's preference, it would be cool to have a 'preferred bpm' so when i sort by bpm it puts the song in the correct preferred bpm

Kudos to every developer out there making this awesome app

**Comment by aqw42** [comment]
Do you mean setting a max / min BPM in bpm analizer configuration ?
example : Between 150 and 200 for jungle, so the track comes out at 170 instead of 85 ?

**Comment by rodrigonsh** [comment]
No, Lets suppose I have this song with 135BPM  (Everglow - Bon Bon Chocolat) but I think it sounds better at 117. 
I mixxx detects 135 but I could "fix" the BPM at 117 and it would show up in my playlist as 117  so when I sort by bpm this 135BPM song would appear next to 116 and 118 BPM songs .... maybe with a little lock icon next to the bpm.

This would be useful because I usually have to open the song in Audacity, change the tempo and save the file as another FLAC named like artist-title(fixedBPM).flac

**Comment by ronso0** [comment]
This sounds interesting, I understand that use case.

Re implementation:
Currently, a BPM search uses the track's BPM value / the value of the 'bpm' column in the database.
To also search for 'preferred BPM' , we'd need a new database column 'preferredBpm' and the Track object would need to have a new 'preferredBpm' property.
Then the search engine could check these two values.

Assuming this is possible, wouldn't it be confusing if you search `bpm:117`, load one of the found tracks (without double-checking the actual BPM) and play it right away, and **then** notice it's actually 135?
